File: app/src/test/java/com/cylan/jiafeigou/n/view/activity/ApSettingActivityTest.java
package com.cylan.jiafeigou.n.view.activity;


import android.widget.TextView;

import com.cylan.jiafeigou.utils.ContextUtils;
import com.cylan.jiafeigou.utils.RandomUtils;

/**
 * Created by hds on 17-9-5.
 */
public class ApSettingActivityTest {


    public void test() {
        TextView tv = new TextView(ContextUtils.getContext());
        tv.setOnClickListener(v -> {
            int r = RandomUtils.getRandom(20);
            if (r > 20) {
                System.out.println("");
                return;
            }
            System.out.println("");
        });
    }
}

File: app/src/test/java/com/cylan/jiafeigou/n/view/activity/ApSettingActivityTest.kt
package com.cylan.jiafeigou.n.view.activity

/**
 * Created by hds on 17-9-8.
 */
class ApSettingActivityTest {

    private val test: Test? = null

    private inner class Test {
        fun action() {}
    }

    private inner class Good {

        private fun test() {
            test!!.action()
        }
    }

}